### DISCUSSION TITLE
Share your presets!

### DISCUSSION THREAD

**Discussion by rerdavies** [rerdavies/pipedal#106] [Share your presets here!]
Share your presets here. Presets can be uploaded to and downloaded from PiPedal using the menu to the right of the Presets dropdown. Download your preset from PiPedal, and attach it by dragging and dropping the file to the area at the bottom of your post. You can also upload and download entire banks of presets, using the _Manage Banks..._ dialog that is accessible from the right-hand hamburger menu. (Upload and download are in the ... menu at the top-right corner).

If you use LV2 plugins in the preset that are not part of the pre-installed set of plugins, don't forget to let everyone know which plugins you used, and which plugin package they came from. The plugins that come pre-installed with PiPedal are minimal set that are intended to get you started. But they are in no way intended to be a complete sets. So don't be shy about including non-default plugins.

**Comment by micahvdm** [comment]
Have you thought about using patchstorage for preset sharing? It has a great API for integration.

**Reply by rerdavies** [reply]
Generated a Feature Request Issue for this. Thanks.

**Comment by maarthome** [comment]
Am I the first one? 😂

I really think that this preset makes my guitar sound very full and spacial... 
[Song1_part1_Git_Voc_v2.zip](https://github.com/user-attachments/files/16803282/Song1_part1_Git_Voc_v2.zip)

Left signal is the guitar and the richt channel is my mike... for recoreding guitar and voice.

You probabely do not have all the plugings, but you can find and download them here: 
https://patchstorage.com/platform/lv2-plugins/

let me know what yout think.

Cheers,
Maarten

Perhaps more humans want to share their favorite presets?

**Comment by bbischler** [comment]
Hi everyone!

Thrilled to share my first PiPedal patch upload to Patchstorage.com: Honeywood Acoustic Guitar!

I created this patch to give an acoustic guitar a warm, full, and genuinely authentic acoustic sound.

Grab it here: https://patchstorage.com/honeywood-acoustic-guitar/

Give it a try and let me know your thoughts!

**Comment by luischos** [comment]
Hi everyone!!
I downloaded and installed this, but I don't know how to import (or where to find) more presets. You have shared some but I can't find them. How can I import them, and where can I download them?
Thanks! 👯

**Reply by FoolHen** [reply]
Hey, here's the official page to share and get presets: https://patchstorage.com/platform/pipedal/

Unfortunately there's only 3 so far but hopefully will increase. I definitely want to share some more 😁

Once you download them you can import the .piPreset files through pipedal UI. From memory its in manage presets -> import preset. You can do that from the phone or the browser. Keep in mind that some presets requiere some specific plugins that you must install too if you don't have them. Those are installed in the Linux command line, not as easy as presets unfortunately. The author of the presets should have written what plugins you need and linked them.

**Comment by StoB3** [comment]
Here's a bank of quickly assembled/vibed presets with ToobAmp LV2 guitar amp plug-ins. The unifying style of these is perhaps experimental psychedelic rock.

[Experimental Psych.zip](https://github.com/user-attachments/files/25870541/Experimental.Psych.zip)

Thank you so much for this wonderful code/software.

Peace, love & harmony to everyone!
